traffic_ops/traffic_ops_golang/region/regions.go

```diff
@@ -116,8 +117,38 @@ JOIN division d ON r.division = d.id ` + where + orderBy + pagination +
 }
 
 func (rg *TORegion) Update(h http.Header) (error, error, int) { return api.GenericUpdate(h, rg) }
-func (rg *TORegion) Create() (error, error, int)              { return api.GenericCreate(rg) }
-func (rg *TORegion) Delete() (error, error, int)              { return api.GenericDelete(rg) }
+
+func (rg *TORegion) Create() (error, error, int) {
+	resultRows, err := rg.APIInfo().Tx.NamedQuery(rg.InsertQuery(), rg)
+	if err != nil {
+		return api.ParseDBError(err)
+	}
+	defer resultRows.Close()
+
+	var divisionName string
+	var id int
+	var lastUpdated tc.TimeNoMod
+
+	rowsAffected := 0
+	for resultRows.Next() {
+		rowsAffected++
+		if err = resultRows.Scan(&id, &lastUpdated, &divisionName); err != nil {
+			return nil, fmt.Errorf("could not scan after insert: %s\n)", err), http.StatusInternalServerError
```

Comment:
Error strings should not contain newlines, also this should wrap `err` using `%w` instead of coercing it to a string with `%s`

Reply:
removed the new line and updated %w.